routes/auth.js

```diff
@@ -202,16 +203,21 @@ module.exports = function(router, passport, async, nodemailer, crypto, models) {
                             return res.status(200).json({info: 'Password reset token is invalid or has expired'});
                         }
                         const user = {email: data.email};
-                        localUser.update({
-                            resetPasswordToken: null,
-                            resetPasswordExpires: null,
-                            password: req.body.password
-                        }, {
-                            where: {
-                                resetPasswordToken: req.params.token,
-                                provider: 'local'
+                        bcrypt.hash(req.body.password, 10, function(err, hash) {
+                            if(err) {
```

Comment:
Add the rule you mentioned and [keyword-spacing](https://eslint.org/docs/rules/keyword-spacing) , It now works fine. Thanks a lot.